### PR TITLE
React JS Free Course Added for Complete Beginners

### PR DESCRIPTION
**Pull request by hallowshaw**
## What does this PR do?
Add resource(s)

### Description
It is a free React JS course for beginners in Hindi by Hitesh Choudhary aka Chai aur Code. This is a very famous emerging playlist. 

### Why is this valuable (or not)?
It is a very detailed course for the beginners and well explained as well. He explains each and every concept while building react projects side by side which in turn helps the learners in knowing how to approach and build a project.

### How do we know it's really free?
It is free because it's on YouTube

### For book lists, is it a book? For course lists, is it a course? etc.
Yes it is a free video course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -365,6 +365,7 @@
 
 * [React JS (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhignjLLXTJWkRJKN-SgAqClL) - Rajesh Kumar, Geeky Shows
 * [React JS Tutorial in Hindi \| React JS for Beginner to Advanced \| Step by Step Video Tutorials](https://www.youtube.com/playlist?list=PLjVLYmrlmjGdnIQKgnTeR1T9-1ltJEaJh) - WsCubeTech
+* [React JS Tutorials for Begineers in Hindi](https://www.youtube.com/watch?v=vz1RlUyrc3w&list=PLu71SKxNbfoDqgPchmvIsL4hTnJIrtige) - Chai aur Code
```
Comment (eshellman) [inline courses/free-courses-hi.md:368]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Comment (hallowshaw) [reply courses/free-courses-hi.md:368]: Please look into it. I have provided playlist link only and did mention the creator - Chai aur Code.

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -365,6 +365,7 @@
 
 * [React JS (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhignjLLXTJWkRJKN-SgAqClL) - Rajesh Kumar, Geeky Shows
 * [React JS Tutorial in Hindi \| React JS for Beginner to Advanced \| Step by Step Video Tutorials](https://www.youtube.com/playlist?list=PLjVLYmrlmjGdnIQKgnTeR1T9-1ltJEaJh) - WsCubeTech
+* [React JS Tutorials for Begineers in Hindi](https://youtube.com/playlist?list=PLu71SKxNbfoDqgPchmvIsL4hTnJIrtige&si=0p_R4-Ci_N0AJoTO) - Chai aur Code
```
Comment (eshellman) [inline courses/free-courses-hi.md:368]:
```suggestion
* [React JS Tutorials for Begineers in Hindi](https://youtube.com/playlist?list=PLu71SKxNbfoDqgPchmvIsL4hTnJIrtige) - Chai aur Code
```
remove tracking code

**Comment by hallowshaw** [comment]
Done

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**